api/src/mailer.ts

```diff
@@ -23,13 +23,20 @@ export default function getMailer(): Transporter {
 			};
 		}
 
+		const tls: Record<string, unknown> = {};
+
+		if (env.EMAIL_SMTP_TLS_CIPHERS) {
+			tls.ciphers = env.EMAIL_SMTP_TLS_CIPHERS;
+		}
+
```

Comment:
We should rely on the getConfig utility here. That'll also ensure we support any and all other configuration options, and maintains consistency with other configuration properties (like storage adapters)

Reply:
Can you give me an example of how to implement getConfig utility? Ty!

Reply:
This is the utility: https://github.com/directus/directus/blob/main/api/src/utils/get-config-from-env.ts

https://github.com/directus/directus/blob/87ab9b3dfa3769e2b391619017f872cc5d3bd53a/api/src/middleware/session.ts#L11

You pass it the prefix of the options you want, and it'll return an object of the env vars (camelcased) that start with that prefix 🙂

Reply:
done for tls config